src/main/steps/common/confirm-contact-details/address/lookup/AddressLookupPostController.ts

```diff
@@ -0,0 +1,12 @@
+import autobind from 'autobind-decorator';
```

Comment:
This file is duplicate to the one which we already have under /applicant section. Can we remove one from /applicant section and use this common file?

Reply:
Will complete refactoring as part of follow up ticket - other refactoring required throughout the different journeys